cartoframes/context.py

```diff
@@ -473,6 +473,67 @@ def sync(self, dataframe, table_name):
         """
         pass
 
+    def fetch(self, query, decode_geom=False, query_columns=None):
+        """Pull the result from an arbitrary SQL query from a CARTO account
+        into a pandas DataFrame.
+
+        Args:
+            query (str): Query to run against CARTO user database. This data
+              will then be converted into a pandas DataFrame.
+            decode_geom (bool, optional): Decodes CARTO's geometries into a
+              `Shapely <https://github.com/Toblerity/Shapely>`__
+              object that can be used, for example, in `GeoPandas
+              <http://geopandas.org/>`__.
+          query_columns (object, optional): The SQLClient `fields` attribute
```

Comment:
Why would a user want to avoid retrieving this? Retrieving them is pretty fast operation (~100ms or less?). We should do some more education about what the `SQLClient` is/where to find it/what a sample argument looks like.

It also looks like this needs two more spaces to be aligned with other docs arguments. And `object` should be replaced with `dict`.

Reply:
mmm now that I read the code again, I think we could just remove the `query_columns`. 

I added it to avoid calling the `table_columns` method twice from the `dataset.download` method, but as you said the query to get the table columns is inexpensive and having that argument there, produces more confusion than anything else.

Thanks for pointing this out :)